tests/cases/conformance/jsx/tsxReactEmitJsxFactory4.tsx

```diff
@@ -0,0 +1,12 @@
+//@filename: file.tsx
+//@jsx: react
+//@jsxFactory: a.b.c

```

Comment:
OK, so we need to verify it is a qualified name. we do not want unexpected characters in there.